toc/working-groups/app-runtime-interfaces.md

```diff
@@ -49,6 +49,8 @@ bots:
   github: capi-bot
 - name: ARI WG Git Bot
   github: ari-wg-gitbot
+- name: CF CLI Eng
+  github: cf-cli-eng
```

Comment:
Is there a real need to add cf-cli-eng as WG bot and not as WG area bot just for the CLI area?
See Notification WG area for an example.

There are more bot users that should be moved from WG into WG area. I will put this topic on the agenda for the next WG meeting.

Reply:
I didn't realize you could scope bots to more specific areas.  Here are the example lines https://github.com/cloudfoundry/community/blob/522836cdbcba75494e25ebfce10e2ef42777ae46/toc/working-groups/app-runtime-interfaces.md?plain=1#L432-L435